lib/LedgerSMB/Scripts/login.pm

```diff
@@ -134,7 +135,7 @@ sub authenticate {
                      [ 'Content-Type' => 'text/plain; charset=utf-8' ],
```

Comment:
We should change the preceding line so that the return string is a constant.
The constant declaration should be something like.
`use constant { HTTP_454_DB_NOT_EXIST => '454 Database Does Not Exist', };`

elsewhere I proposed adding `HTTP_PROHIBITED_USERNAME => '454'` in light of this additional use of 454 I think that should change to be....
`use constant { HTTP_454_PROHIBITED_USERNAME => '454 Prohibited Username', };`

Any other uses of 454 should then follow the same pattern, unique constants declared for each type of error message.